alternator/executor.cc

```diff
@@ -7,6 +7,7 @@
  */
 
 #include <fmt/ranges.h>
+#include <seastar/core/on_internal_error.hh>
```

Comment:
In the commit message you have _yet_ repetition in "It is not yet wired to anything yet".